borgcron.sh

```diff
@@ -173,7 +182,8 @@ for i in $( seq "$REPEAT_NUM" ); do
 
 	# add local lock
 	do_lock
-
+	track_exitcode "$?"
```

Comment:
You do this to check the exit code of `do_lock`? That's not necessary, `do_lock` will force an exit if it really fails and cannot lock (for safety reasons), so you can remove this line.

Reply:
✅